docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+# Deploying Wordpress to GKE
+
+This user guide will walk you through Wordpress application deployment using
+your existing Kubernetes cluster and Crossplane managed resources. We will:
+* Install Crossplane to your cluster.
+* Create necessary resource classes for MySQL database instance.
+* Create network resources to get GKE cluster to connect to MySQL instance.
+* Deploy Wordpress.
+
+
+## Pre-requisites
+
+* [gcloud CLI](https://cloud.google.com/sdk/docs/quickstarts)
+    * Make sure to configure the authentication after installing gcloud cli.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+
+## Preparation
+This guide assumes that you have setup the gcloud CLI and are logged in to your
+desired account.
+
+*Note: environment variables are used throughout this guide. You should use
+your own values.*
+
+Run the following:
+```bash
+export PROJECT_ID=crossplane-playground # the project that all resources reside.
+export NETWORK_NAME=default # the network that your GKE cluster lives in.
+export SUBNETWORK_NAME=default # the subnetwork that your GKE cluster lives in.
+``` 
+
+### GKE Cluster
+We are assuming you've got a GKE cluster up and running created either via 
+GCP Console or gcloud CLI. If you don't, the basic command to create one is
+the following:
+
+1. Create a GKE cluster:
+```bash
+gcloud beta container --project "${PROJECT_ID}" clusters create "my-existing-cluster" \
+  --zone "us-central1-a" \
+  --cluster-version "1.13.7-gke.8" \
+  --machine-type "n1-standard-1" \
+  --image-type "COS" \
+  --disk-type "pd-standard" \
+  --disk-size "100" \
+  --metadata disable-legacy-endpoints=true \
+  --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" \
+  --num-nodes "1" \
+  --enable-cloud-logging --enable-cloud-monitoring --enable-ip-alias --enable-autoupgrade --enable-autorepair --no-enable-basic-auth \
+  --network "projects/${PROJECT_ID}/global/networks/${NETWORK_NAME}" \
+  --subnetwork "projects/${PROJECT_ID}/regions/us-central1/subnetworks/${SUBNETWORK_NAME}" \
+  --default-max-pods-per-node "110" \
+  --addons HorizontalPodAutoscaling,HttpLoadBalancing
+```
+
+This may take a while to finish.
+
+4. Connect to your GKE cluster
+```bash
+gcloud container clusters get-credentials "my-existing-cluster" --zone us-central1-a --project "${PROJECT_ID}"
```

Comment:
How does this set up a connection? Does it automatically populate our `kubectl` config for us and switch to it?

Reply:
> Does it automatically populate our kubectl config for us and switch to it?

That's exactly what it does.